CHANGELOG.rst

```diff
@@ -1,5 +1,8 @@
 Changelog for Pulse3D
 =====================
+0.25.0 (2022-23-22)
```

Comment:
2022-08-23

Reply:
I keep forgetting that the yyyy-mm-dd is standard not yyyy-dd-mm